crates/iota-types/src/iota_system_state/iota_system_state_summary.rs

```diff
@@ -71,11 +71,15 @@ pub struct IotaSystemStateSummary {
     #[schemars(with = "BigInt<u64>")]
     #[serde_as(as = "Readable<BigInt<u64>, _>")]
     pub safe_mode_storage_charges: u64,
-    /// Amount of computation rewards accumulated (and not yet distributed)
+    /// Amount of computation charges accumulated (and not yet distributed)
     /// during safe mode.
     #[schemars(with = "BigInt<u64>")]
     #[serde_as(as = "Readable<BigInt<u64>, _>")]
-    pub safe_mode_computation_rewards: u64,
+    pub safe_mode_computation_charges: u64,
+    /// Amount of burned computation charges accumulated during safe mode.
+    #[schemars(with = "BigInt<u64>")]
+    #[serde_as(as = "Readable<BigInt<u64>, _>")]
+    pub safe_mode_computation_charges_burned: u64,
```

Comment:
This breaks backwards compatibility for the JSON-RPC API (both the rename and the addition of the new field).

If we could version this as well, it would allow more flexibility in handling versioning in the API as well.

An idea would be to restructure the parent module to
```sh
iota_system_state/
   v1/
   v2/
```

Another would be to split `iota_system_state_summary` into `iota_system_state_summary::{v1,v2}`.

And last we could introduce a new `IotaSystemStateSummaryV2`.

Reply:
Introducing a V2 would require an enum like we have for IotaSystemState as well, and changing IotaSystemStateTrait and quite a lot of other code. Is a new version the only way we can change types in the JSON-RPC API in general, just like with the move types and their rust versions?